### PR TITLE
Fix always false

### PR DESCRIPTION
**Pull request by raniervf**
The part expression is always false.

### CODE REVIEW

**Comment by dinhviethoa** [comment]
This one is a little bit different in style from the other...

**Comment by raniervf** [comment]
Is ok, sent fixes with pull request? 
Or there are way to sent more patchs?
Maybe with git diff (local clone libetpan)?

regards,
Ranier Vilela

________________________________________
De: Hoà V. DINH <notifications@github.com>
Enviado: quarta-feira, 29 de abril de 2020 23:50
Para: dinhviethoa/libetpan
Cc: Ranier Vilela; Author
Assunto: Re: [dinhviethoa/libetpan] Fix always false (#350)

This one is a little bit different in style from the other...

**Comment by dinhviethoa** [comment]
pull requests are perfect! You can update this one!
Was there any issue with the original code besides the indentation?

**Comment by raniervf** [comment]
De: Hoà V. DINH <notifications@github.com>
Enviado: quinta-feira, 30 de abril de 2020 00:09
Para: dinhviethoa/libetpan
Cc: Ranier Vilela; Author
Assunto: Re: [dinhviethoa/libetpan] Fix always false (#350)

>pull requests are perfect! You can update this one!
>Was there any issue with the original code besides the indentation?
The problem is that i am editing with the browser.
Is it possible to make a patch with git diff and attach it to the pull request?

regards,
Ranier Vilela

File: src/low-level/imap/mailimap_parser.c [hunk 1]
```diff
@@ -6017,14 +6017,16 @@ static int mailimap_literal_parse_progress(mailstream * fd, MMAPString * buffer,
...
   r = mailimap_crlf_parse(fd, buffer, parser_ctx, &cur_token);
-	if (r == MAILIMAP_ERROR_PARSE) {
-		/* workaround for Lotus Domino IMAP server */
-		mailimap_space_parse(fd, buffer, &cur_token);
-		mailimap_space_parse(fd, buffer, &cur_token);
-	}
-	else if (r != MAILIMAP_NO_ERROR) {
-    res = r;
-    goto err;
+  if (r != MAILIMAP_NO_ERROR) {
```
Comment (dinhviethoa) [inline src/low-level/imap/mailimap_parser.c:6020]: Could you describe what you are fixing here?
The original code seemed fine.

Comment (raniervf) [reply src/low-level/imap/mailimap_parser.c:6020]: The alert is the part expression 'r != MAILIMAP_NO_ERROR' is always false.
It seems a little confused, as it is.
I believe it was more readable.

**Pull request closed without merging**